### DISCUSSION TITLE
Route unhandled bounces?

### DISCUSSION THREAD

**Discussion by gazugafan** [postalserver/postal#2138] [Help with using Postal] [answered]
If a message comes to the return-path, but Postal can't determine which message it relates to, can the message be routed somewhere as a fallback? I see this in Postal's event log for the unhandled bounce...

```This message was a bounce but we couldn't link it with any outgoing message and there was no route for it.```

... but how would I setup a route for unhandled bounces?

**Comment by willpower232** [comment, marked as answer]
There is `__returnpath__` but I don't think that is specifically for unhandled bounces

https://docs.postalserver.io/other/auto-responders-and-bounces

**Reply by gazugafan** [reply]
Oh, perfect! According to the docs, ```__returnpath__``` *is* specifically for unhandled bounces...

> This is the same as a normal route except you should enter the name as ```__returnpath__``` and leave the domain field blank. You can only choose HTTP endpoints for this type of route. Once added, messages sent to your return path that aren't detected as bounces will be sent to HTTP endpoint you choose.

... kind've weird that you can only use HTTP endpoints, but that's okay. I'll give this a shot today. Thanks!!
